<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Membros</class>
 <widget class="QDialog" name="Membros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cupcake Store - membros</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursoss.qrc">
    <normaloff>:/Imagens/imgs/th.png</normaloff>:/Imagens/imgs/th.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;
	font-size: 16px;
	background: white;
}

QToolButton
{
	background: transparent;
	border-radius: 40px;
}

QPushButton
{
	background: green;
	border-radius: 40px;
}

Qlabel
{
	color: white;
	background: transparent;
}

QPushButton
{
	color: black;
	border-radius:15px;
}

QPushButton:hover
{
	color: #333;
	border-radius:15px;
	background: #49ebff;
}

QLineEdit
{
	background: green;
	border: none;
	color: black;
	border-bottom: 1px solid white;
}

</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_novoMembro">
    <attribute name="title">
     <string>Novo membro</string>
    </attribute>
    <widget class="QPushButton" name="btn_addMembro">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar membro</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>51</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/th.png</normaloff>:/Imagens/imgs/th.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>270</y>
       <width>51</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/th.png</normaloff>:/Imagens/imgs/th.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>32</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>88</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Celular:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>145</y>
       <width>127</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Novo username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>201</y>
       <width>98</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nova senha:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>261</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="txt_nome"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_celular"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_user"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_senha">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_membros">
    <attribute name="title">
     <string>Membros atuais</string>
    </attribute>
    <widget class="QTableWidget" name="tab_listaMembros">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>231</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>241</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_nomeMembro"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Celular</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_celularMembro"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_userMembro"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_senhaMembro">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_alterarMembro">
          <property name="text">
           <string>Alterar</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_excluirMembro">
          <property name="text">
           <string>Excluir</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>270</y>
       <width>51</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="recursoss.qrc">
       <normaloff>:/Imagens/imgs/th.png</normaloff>:/Imagens/imgs/th.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txt_nome</tabstop>
  <tabstop>txt_celular</tabstop>
  <tabstop>txt_user</tabstop>
  <tabstop>txt_senha</tabstop>
  <tabstop>btn_addMembro</tabstop>
  <tabstop>tab_listaMembros</tabstop>
  <tabstop>txt_nomeMembro</tabstop>
  <tabstop>txt_celularMembro</tabstop>
  <tabstop>txt_userMembro</tabstop>
  <tabstop>txt_senhaMembro</tabstop>
  <tabstop>btn_alterarMembro</tabstop>
  <tabstop>btn_excluirMembro</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>toolButton_3</tabstop>
 </tabstops>
 <resources>
  <include location="recursoss.qrc"/>
 </resources>
 <connections/>
</ui>
